src/migrations/versions/295b9b40e54d_seed_privacy_request_data.py

```diff
@@ -0,0 +1,124 @@
+"""seed privacy request data
```

Comment:
We'll need to remove this file again before we merge, since this migration was never merged into `main` on the core repo.